src/main/app/src/assets/i18n/en.json

```
@@ -759,6 +759,8 @@
   "msg.error.besluit.publication.date.missing": "Missing decision publication date",
   "msg.error.besluit.response.date.missing": "Missing decision response date",
   "msg.error.besluit.response.date.invalid": "Response date for decision should not be before publication date",
+  "msg.error.case.has.open.subcases": "This case has open subcases and can not be closed",
```

```suggestion
  "msg.error.case.has.open.subcases": "This case has open subcases and cannot be closed",
```

https://www.merriam-webster.com/grammar/cannot-vs-can-not-is-there-a-difference ;-)

---

And: close the sentence with a `.` or not?